test/XRP/fakes/fake-xrp-protobufs.ts

```diff
@@ -467,6 +500,127 @@ testInvalidPaymentTransaction.setSigningPublicKey(signingPublicKeyProto)
 testInvalidPaymentTransaction.setTransactionSignature(transactionSignatureProto)
 testInvalidPaymentTransaction.setPayment(testInvalidPaymentProtoNoAmount)
 
+// ... no account
```

Comment:
extremely great job on all these invalid test cases.  I'm going to be annoying and ask for one more:  can you make a fake transaction proto that's fine with the exception of a bad `Payment` as its `transaction` field?  I know we have a bad payment protobuf just laying around so hopefully it's simple enough.  It'll get us codecov on the one missing line.

Reply:
It won't, because a bad `Payment` will trigger an error in `XrpPayment`. The one missing line is unreachable, but I can't think of a way to get rid of it, because as far as Typescript is aware, `payment` can still be `undefined`, even though the protobuf object won't let that be true when the data case is `Payment`.

Reply:
Ohhh I see.  I guess one move would be to remove that check entirely because it probably is an impossible case logically, but that seems less defensive and unnecessary.  We can take the codecov hit

Reply:
Unfortunately Typescript gets mad if it's removed, since it could be `undefined`, so I don't think it can be removed.